### PR TITLE
Writing_Good_Commits.rst: Reword

### PR DESCRIPTION
**Pull request by ioanavoica**
Fixes https://github.com/coala/documentation/issues/149

### CODE REVIEW

**Comment by tulphoon** [comment]
Please squash your commits, here's a link which explains the process. http://docs.coala.io/en/latest/Developers/Git_Basics.html#squashing-your-commits

File: Developers/Writing_Good_Commits.rst [hunk 1]
```diff
@@ -32,9 +32,8 @@ At coala we are looking heavily at the maintainability of the code.
 
     Code is more often read than written!
 
-It is obvious that we need good code. In order to do that we are
-verifying that every change to our code (i.e. the commits) is making it
-better.
+It is obvious that we need good code. For this, we make sure that every change
+ (i.e. the commits) makes our code better.
```
Comment (tulphoon) [inline Developers/Writing_Good_Commits.rst]: the space before `(i.e. the commits) is unnecessary, remove it

**Comment by tulphoon** [comment]
ack c528743

**Comment by sils** [comment]
Hi,

it looks like your PR is out of date and needs a rebase. [This page](http://coala.io/git) may help you to get started on this. (*Do not press the update branch button. It's evil.*)

Add the original coala repository as a remote:

```
git remote add upstream https://github.com/coala-analyzer/coala.git
```

(Be sure to use coala-bears on the coala-bears repository :))

Then go to your branch and rebase it onto our master:

```
git checkout <yourbranch>
git fetch upstream
git rebase --interactive upstream/master
```

Now push it up. You will need to do a force push because you change history here (do not do this on master branches :)):

```
git push --force
```

If you're just looking to fix an issue very quickly and not interested in contributing to coala long term, we can fix it up for you - just tell us :).

Ping us on [our gitter channel](https://coala.io/chat) if you need help!

**Comment by Adrianzatreanu** [comment]
closing due to high inactivity

**Pull request closed without merging**